### PR TITLE
Fix setting locale code

### PR DESCRIPTION
**Pull request by KazuCocoa**
ref: https://github.com/appium/appium-android-driver/pull/287

After merging ^ and publishing the new version, I'll update `"appium-android-driver": "^1.27.1",` in `package.json`.

### CODE REVIEW

**Comment by imurchie** [comment]
The latest android driver has been published as `appium-android-driver@1.34.0`.

**Comment by KazuCocoa** [comment]
📝 
Failing test will fix after https://github.com/appium/appium-adb/pull/282

File: test/functional/commands/strings-e2e-specs.js [hunk 1]
```diff
@@ -37,15 +37,17 @@ describe('strings', function () {
...
-      // restarting doesn't work on Android 7+
       adb = new ADB();
-      if (await adb.getApiLevel() > 23) return this.skip(); //eslint-disable-line curly
-
       initialLocale = await getLocale(adb);
     });
     afterEach(async function () {
       if (driver) {
-        await androidHelpers.ensureDeviceLocale(adb, null, initialLocale);
+        if (await adb.getApiLevel() > 23) {
```
Comment (KazuCocoa) [inline test/functional/commands/strings-e2e-specs.js]: ✅ succeeded https://travis-ci.org/appium/appium-uiautomator2-driver/jobs/314251450

**Comment by KazuCocoa** [comment]
@imurchie @mykola-mokhnach @dpgraham 
Could someone check this PR?
This PR include updating `appium-adb` and `appium-android-driver`.

File: test/functional/commands/language-e2e-specs.js [hunk 2]
```diff
@@ -10,22 +10,25 @@ import { getLocale } from '../helpers/helpers';
...
     initialLocale = await getLocale(adb);
   });
 
   let driver;
   after(async function () {
     if (driver) {
-      await androidHelpers.ensureDeviceLocale(driver.adb, null, initialLocale);
-
+      if (await adb.getApiLevel() > 23) {
+        let split_locale = initialLocale.split("-");
```
Comment (mykola-mokhnach) [inline test/functional/commands/language-e2e-specs.js]: please use camelCase style for variable naming

Comment (mykola-mokhnach) [reply test/functional/commands/language-e2e-specs.js]: you can use `let [language, country] = initialLocale.split("-")` style

File: test/functional/commands/strings-e2e-specs.js [hunk 3]
```diff
@@ -37,15 +37,17 @@ describe('strings', function () {
...
+          let split_locale = initialLocale.split("-");
```
Comment (mykola-mokhnach) [inline test/functional/commands/strings-e2e-specs.js]: same comment as above

**Pull request merged**